lib/mwlib/src/MW/Media/Image/Imagick.php

```diff
@@ -100,8 +100,14 @@ public function save( $filename = null, $mimetype = null )
 	 */
 	public function scale( $width, $height, $fit = true )
 	{
-		if( $fit === true )
-		{
+		if( $fit === false && $width && $height ) {
```

Comment:
Use `$fit == false` because you didn't cast the value to a boolean before

Reply:
Okay, I will change it.